eclipse-scout-core/src/form/fields/listbox/ListBox.ts

```diff
@@ -83,34 +131,31 @@ export class ListBox<TValue> extends LookupBox<TValue> implements ListBoxModel<T
     this._syncValueToTable(this.value);
   }
 
-  protected _syncValueToTable(newValue: TValue[]) {
+  protected _isEmpty(value: TValue): boolean {
+    return arrays.empty(value as TLookup[]);
+  }
+
+  protected _resetTable() {
```

Comment:
This method seems to be called only once - necessary?

Reply:
it's offered as an extension point